### ISSUE TITLE
xml package dependency upgrade

### ISSUE THREAD

**Issue by rraksi** [ali-thowfeek/simple_html_css_flutter#2]
Please upgrade the `xml` package dependency to at least 4.2.0 because it blocks `flutter` 1.18+ upgrades if used together with `flutter_svg` package.

`flutter_svg` upgrade is needed because of flutter api deprecation: https://github.com/flutter/flutter/pull/58635

```
Because no versions of simple_html_css match >2.0.0+2 <3.0.0 and simple_html_css 2.0.0+2 depends on xml ^3.6.1, simple_html_css ^2.0.0+2 requires xml ^3.6.1.
And because flutter_svg >=0.18.1 depends on xml ^4.2.0, simple_html_css ^2.0.0+2 is incompatible with flutter_svg >=0.18.1.
So, because mobile depends on both flutter_svg ^0.19.1 and simple_html_css ^2.0.0+2, version solving failed.
pub get failed (1; So, because mobile depends on both flutter_svg ^0.19.1 and simple_html_css ^2.0.0+2, version solving failed.)
```

**Comment by ali-thowfeek** [comment]
I've updated the dependencies and published a minor version update (2.0.0+3) Please check and let me know if your problem is resolved.

**Comment by rraksi** [comment]
Thanks for the quick reaction, the update solved the problem.
